platform/ios/scripts/package.sh

```diff
@@ -84,6 +84,7 @@ xcodebuild \
     CURRENT_SHORT_VERSION=${SHORT_VERSION} \
     CURRENT_SEMANTIC_VERSION=${SEM_VERSION} \
     CURRENT_COMMIT_HASH=${HASH} \
+    ONLY_ACTIVE_ARCH=NO \
```

Comment:
We should revisit whether this flag is necessary after mapbox/mason#181 lands. Ideally we’d stick to just the standard architectures, to minimize the build size.